src/driver/mpas_subdriver.F

```diff
@@ -109,38 +111,68 @@ end subroutine xml_stream_get_attributes
       readNamelistArg = .false.
       readStreamsArg = .false.
 
-      nArgs = command_argument_count()
-      iArg = 1
-      do while (iArg < nArgs)
-         call get_command_argument(iArg, argument)
-         if (len_trim(argument) == 0) exit
-
-         if ( trim(argument) == '-n' ) then
-            iArg = iArg + 1
+      ! If provided, error check the namelistFileParam and copy it to namelistFile to override default
+      if (present(namelistFileParam)) then
+         if (len_trim(namelistFileParam) <= 0) then
+            write(0,'(A)') 'WARNING: mpas_init argument ''namelistFileParam'' has 0 or negative length and will be ignored'
+         else if (len_trim(namelistFileParam) > len(namelistFile)) then
+            write(0,'(A,I5,A,I5,A)') 'CRITICAL ERROR: mpas_init argument ''namelistFileParam'' has length ', len_trim(namelistFileParam),&
```

Comment:
Let's wrap this line to ensure it's at most 132 columns (and similarly for the other `CRITICAL ERROR` write statement. (I do realize that there are other lines in this source file that exceed 132 column, but they got in before we decided that standards compliance was a good thing; we can fix those other lines another time...).

Thus spake the Fortran 2003 standard:
```
In free source form there are no restrictions on where a statement (or portion of a statement) may
appear within a line. A line may contain zero characters. If a line consists entirely of characters of
default kind (4.4.4), it may contain at most 132 characters. If a line contains any character that is not
of default kind, the maximum number of characters allowed on the line is processor dependent.
```

Reply:
To avoid too many force pushes, would the below be an acceptable re-write? In case the formatting doesn't show correctly, I'm trying to align `len_trim` with the first quote mark before `CRITICAL`.

>              write(0,'(A,I5,A,I5,A)') 'CRITICAL ERROR: mpas_init argument ''namelistFileParam'' has length ',&
>                                       len_trim(namelistFileParam), ', but the maximum allowed is ', len(namelistFile), ' characters'

Reply:
That seems fine to me. (I'm not sure if GitHub changed the indentation or if I copied the text incorrectly, but as long as we're indented three columns within the `else if`, that should work.)